chain/client/src/client.rs

```diff
@@ -2467,7 +2467,10 @@ impl Client {
             assert_eq!(sync_hash, state_sync_info.epoch_tail_hash);
             let network_adapter = self.network_adapter.clone();
 
-            let shards_to_split = self.get_shards_to_split(sync_hash, &state_sync_info, &me)?;
+            // I *think* this is not relevant anymore, since we download
+            // already the next epoch's state.
+            // let shards_to_split = self.get_shards_to_split(sync_hash, &state_sync_info, &me)?;
+            let shards_to_split = HashMap::new();
```

Comment:
can shards_to_split be removed at all from this function then?

Reply:
Is it fine if we keep this change for now and remove it as part of a future PR?

Reply:
That's what I plan to do, yes. This line is not triggered now anyway.